lib/logstash/codecs/avro.rb

```diff
@@ -6,17 +6,22 @@
 require "logstash/event"
 require "logstash/timestamp"
 require "logstash/util"
+require 'logstash/plugin_mixins/ecs_compatibility_support'
+require 'logstash/plugin_mixins/ecs_compatibility_support/target_check'
+require 'logstash/plugin_mixins/validator_support/field_reference_validation_adapter'
+require 'logstash/plugin_mixins/event_support/event_factory_adapter'
+require 'logstash/plugin_mixins/event_support/from_json_helper'
```

Comment:
:scissors: we do need the `require 'logstash/plugin_mixins/event_support/from_json_helper'`